README.md

```diff
@@ -11,12 +11,34 @@
 - CSS
 - Linters
 
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Code editor
+
+### Setup
+
+git clone the project and open the index.html file!
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes git clone URL , cd project_name etc.

- [ OPTIONAL] Kindly add a meaningful project description to your README file, Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and its essential qualities. Think of it as the elevator pitch that focuses on what and why without delving into how.